packages/react-native-sdk/src/components/LocalVideoView.tsx

```diff
@@ -56,29 +63,59 @@ export const LocalVideoView = (props: LocalVideoViewProps) => {
   const isCameraOnFrontFacingMode = useStreamVideoStoreValue(
     (store) => store.isCameraOnFrontFacingMode,
   );
+  const pan = useRef(new Animated.ValueXY()).current;
+  const panResponder = useRef(
+    PanResponder.create({
+      onMoveShouldSetPanResponder: () => true,
+      onPanResponderMove: Animated.event([null, { dx: pan.x, dy: pan.y }], {
+        useNativeDriver: false,
```

Comment:
why is it false here? is true not supported for this usecase?

false is potentially slow on medium/low end android devices

Reply:
translate animations are supported with useNativeDrive afaik

Reply:
they are supported, it just failed due to an unknow reason now it works 👍🏻

Reply:
https://github.com/facebook/react-native/issues/13377#issuecomment-292657541